docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -462,6 +511,35 @@ After training, the model must be destroyed with `ml_train_model_destroy()`.
 `ml_train_model_add_layer()`, `ml_train_set_optimizer()`, and `ml_train_set_dataset()` transfers ownership to the model.
 `layers`, `optimizers` and `dataset` that belongs to the `model` are also deleted.
 
+## Using the trained model for the inference
+
+The trained model can be used for inference with [Machine Learning Inference API](machine-learning-inference.md).
+
+Ensure that the INI file contains the correct weight file in `save_path`.
+
+For example, since Tizen 6.5, you can use the trained model with single API as follows:
```

Comment:
Is the line spacing after each sentence intentional?
Cant we write like this? 
```suggestion
The trained model can be used for inference with [Machine Learning Inference API](machine-learning-inference.md). Ensure that the INI file contains the correct weight file in `save_path`. For example, since Tizen 6.5, you can use the trained model with single API as follows:

Reply:
Changed as suggested.